packages/create-app/templates/default-app/packages/app/src/components/Root/Root.tsx

```diff
@@ -75,6 +76,7 @@ export const Root = ({ children }: PropsWithChildren<{}>) => (
         <SidebarItem icon={ExtensionIcon} to="api-docs" text="APIs" />
         <SidebarItem icon={LibraryBooks} to="docs" text="Docs" />
         <SidebarItem icon={CreateComponentIcon} to="create" text="Create..." />
+        <NotificationsSidebarItem />
```

Comment:
I'm wondering about the position of this: does it make sense to be here or should we do like @drodil did for the Demo site? https://demo.backstage.io/

Code example:

https://github.com/backstage/demo/blob/846e2be70369053b50e2bdec42df68449e2aff32/packages/app/src/components/Root/Root.tsx#L104-L106

Reply:
Np, done:

![Screenshot from 2024-09-24 09-26-00](https://github.com/user-attachments/assets/2fddb682-39ff-4008-a507-d62c91a50201)